### ISSUE TITLE
Exit but WRONG 0

### ISSUE THREAD

**Issue by CXang** [biofuture/Ublastx_stageone#69]
when I want to get the arg aboundance in my sample ,I find that I always get the situation,like this "EXIT 25",but I get all the output files.when i view the nohup file,there is no wrong.I'm not sure if this will have an effect on the abundance of resistance genes in my sample?
[1]+  Exit 25                 nohup sh scr_arg_num.sh > scr_arg_num.sh.nohup
scr_arg_num.sh.nohup:0 wrong;

**Comment by Bro-Miao** [comment]
I also had the same problem. Did you solve it?

**Comment by CXang** [comment]
Yes,may be you don't install R in your linux,so you got the Exit message.
do you got the results of your sample(like the picture display)?in other words,you got the all the results from the software,but program running result display Exit




---- Replied mail ----
| From | ***@***.***> |
| Date | 02/28/2022 15:59 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [biofuture/Ublastx_stageone] Exit but WRONG 0 (Issue #69) |

I also had the same problem. Did you solve it?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Bro-Miao** [comment]
I didn't get any results and it showed wrong0, and I have R on my workstation that I can use directly. So I am very confused.

**Comment by CXang** [comment]
You may have encountered other problems, you can check the log file for specific errors，You can take a screenshot and send it to me. Another point is that you may need to configure some software(like diamond and usearch,I display in the picture,/path/to/yoursoftware/Ublastx_stageone_master/bin). I am not sure about your specific software version.
I hope to help you,you could give me more information~




---- Replied mail ----
| From | ***@***.***> |
| Date | 03/01/2022 15:11 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [biofuture/Ublastx_stageone] Exit but WRONG 0 (Issue #69) |

I didn't get any results and it showed wrong0, and I have R on my workstation that I can use directly. So I am very confused.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by biofuture** [comment]
Dear all, 
If you use the singularity version of the pipeline.  It should include all the dependent software and scripts.  You need to confirm that you have full access to singularity and an updated version of Linux. 

Another part you can check is the format of the meta data,  please make sure your meta data file follows the format requirements, and especially if you edit your meta data under  Windows with Notepad++ check the end of the meta data, windows is different from Linux/Unix.
